src/routes/verify/verify.ts

```diff
@@ -11,18 +12,29 @@ export const verifyHandler = async (
   const type = req.query.type as string;
   const redirectTo = req.query.redirectTo as string;
 
+  if (!redirectTo) {
+    return res.boom.badRequest('Missing redirectTo');
+  }
+
   if (!ticket) {
-    return res.boom.badRequest('Missing ticket');
+    const redirectUrl = generateRedirectUrl(redirectTo, {
+      error: 'missingVerificationTicket',
```

Comment:
Given that this is considered an object or type, would it make sense to capitalize the first letter (`MissingVerificationTicket`) ?